chapter-customizing.asciidoc

```diff
@@ -1212,7 +1212,8 @@ normal part of the Maven lifecycle. You run Maven tests whenever you
 run +mvn package+ or +mvn install+. If you would like to run all the
 lifecycle phases up to and including the +test+ phase, run +mvn test+:
 
-----
+[listing]
```

Comment:
Changed this block quote style because the lines of dashes inside were being parsed as block quote delimiters and breaking it up in to different sections, with "T E S T S" being rendered as normal body text, which looked like a subsection title.